### DISCUSSION TITLE
Is there anyway to start other container app without reset all container?

### DISCUSSION THREAD

**Discussion by jupitert25** [engageub/InternetIncome#67] [Q&A]
Hi .  I run honeygain with multi proxy each folder and now I want to add other app in each folder but when i use command delete and start honeygain won't connect unlti I change device name so honeygain will keep stacking  device slots
So is there anyway to start another app without reset the whole script?

**Comment by engageub** [comment]
Hello,
Frankly speaking, I did not completely understand what you were trying to convey.
What I understand is that you are trying to use multiple accounts.
Each honeygain account has a limit of 10 devices.
You can start 10 devices in 1 folder by using proxies in proxies.txt file.
Honeygain may use device name for identification. 
If you are using different folders but with same account, you have to change the device name in properties.conf in each folder.

Thank you

**Reply by jupitert25** [reply]
yes . I didn't know I can run 10 device in 1 folder .  
I only run 1 account but set each folder with each proxy . 
I guess next time I will setting like you said
what about add other app without reset?

**Reply by engageub** [reply]
You can run all the apps from same folder using multiple proxies but with one account in each folder.
You have to delete using --delete and add new app if you wish to use in same folder. 

Thank you
